subsys/bluetooth/mesh/adv.c

```diff
@@ -52,13 +52,20 @@ K_MEM_SLAB_DEFINE_STATIC(local_adv_pool, sizeof(struct bt_mesh_adv),
 #if defined(CONFIG_BT_MESH_RELAY_BUF_COUNT)
 K_MEM_SLAB_DEFINE_STATIC(relay_adv_pool, sizeof(struct bt_mesh_adv),
 			 CONFIG_BT_MESH_RELAY_BUF_COUNT, __alignof__(struct bt_mesh_adv));
+#elif defined(CONFIG_BT_MESH_BRG_BUF_COUNT)
+K_MEM_SLAB_DEFINE_STATIC(relay_adv_pool, sizeof(struct bt_mesh_adv),
+			 CONFIG_BT_MESH_BRG_BUF_COUNT, __alignof__(struct bt_mesh_adv));
 #endif
 
 #if defined(CONFIG_BT_MESH_ADV_EXT_FRIEND_SEPARATE)
 K_MEM_SLAB_DEFINE_STATIC(friend_adv_pool, sizeof(struct bt_mesh_adv),
 			 CONFIG_BT_MESH_FRIEND_LPN_COUNT, __alignof__(struct bt_mesh_adv));
 #endif
 
+#if (!defined(CONFIG_BT_MESH_RELAY_ADV_SETS) && defined(CONFIG_BT_MESH_BRG_ADV_SETS))
+#define CONFIG_BT_MESH_RELAY_ADV_SETS CONFIG_BT_MESH_BRG_ADV_SETS
+#endif
```

Comment:
Not sure if this is the best way to go about it, but added this to simplify the expression following line 212